public/video-ui/src/components/Video/Display.js

```diff
@@ -41,6 +41,7 @@ class VideoDisplay extends React.Component {
 
   updateVideo = video => {
     this.props.videoActions.updateVideo(video);
+    return Promise.resolve();
```

Comment:
why not return `this.props.videoActions.updateVideo(video);`?

Reply:
Because it doesn't return a promise - it's not actually saving anything to the database, only to the redux store.